src/constants/default-settings.js

```diff
@@ -185,8 +186,7 @@ export const THEME = keyMirror({
  * ```
  */
 export const LOCALES = keyMirror({
```

Comment:
you are duplicating the emptyLocales variable. You don't need to create a new variable __LOCALES__ since you already have emptyLocales.
Simply import __emptyLocales__ where you need it

Reply:
if you are no longer using emptyLocales, what is gonna happen to LOCALES?

Reply:
We will use localeCodes instead, please check last commit